sycl/doc/extensions/experimental/sycl_ext_oneapi_complex.asciidoc

```diff
@@ -54,10 +54,20 @@ specification.*
 While {dpcpp} has support for `std::complex` in device code, it limits the
 complex interface and operations to the existing C++ standard. This proposal
 defines a SYCL complex extension based on but independent of the `std::complex`
-interface. This framework would allow for further development of complex math
-within oneAPI. Possible areas for deviation with `std::complex` include adding
-complex support for `marray` and `vec` and overloading mathematical
-functions to handle the element-wise operations.
+interface.
+
+The proposed framework not only encompasses complex support for traditional use
+cases but also accommodate for advanced mathematical features and data
+structures.
+
+Specifically, we propose to incorporate complex support for `sycl::marray`.
+This addition will empower developers to store complex numbers seamlessly
+within arrays, opening up new possibilities for data manipulation and
```

Comment:
```suggestion
within a `sycl::marray`, opening up new possibilities for data manipulation and
```

Is this what you meant?  Developers can already create a `std::array` of `std::complex`, right?

Reply:
- This is what I meant with the "array", but clearly, it is not explicit enough and creates confusion.
- Correct
- I'll apply the suggestion :+1: